src/Core/src/Handlers/ShapeView/ShapeViewHandler.Windows.cs

```diff
@@ -20,7 +20,10 @@ public static void MapBackground(IShapeViewHandler handler, IShapeView shapeView
 			{
 				handler.UpdateValue(nameof(IViewHandler.ContainerView));
 				handler.ToPlatform().UpdateBackground(shapeView);
+			}
 
+			if (shapeView.Background is not null || shapeView.Fill is not null)
```

Comment:
Can we move this whole method into the shared file? 

It seems like this method is identical between all three platforms

Reply:
InvalidateShape is an extension method for platform specific class, or maybe you meant something else?
<img width="981" alt="Screenshot 2024-11-25 at 00 58 34" src="https://github.com/user-attachments/assets/f5375c18-ff13-4daa-a93c-23a90ab34df0">